app/src/main/dist/config/crate.yml

```diff
@@ -58,6 +58,74 @@
 # features of the Enterprise Edition is governed by the terms and conditions of
 # your Enterprise or Subscription Agreement with Crate.io.
 
+############################ JMX Monitoring Plugin ############################
```

Comment:
I would do something like
#############  JMX Monitoring Plugin - Enterprise Version only ##############
and same for the rest, or some how distinguish the Enterprise features.
@nslater what do you say?